cli/src/factory_ops/unboxing/config_update

```diff
@@ -99,16 +99,20 @@ function update_ssh_config {
 function update_salt_minion {
 
     if [[ "srvnode-1" == "$(cat /etc/salt/minion_id)" ]]; then
-        local host=$(hostname)
+        # Getting '127.0.0.1'. This can be hard-coded also
+        local localhost_ip=$(awk '{print $1; exit}' /etc/hosts)
+        local primary_host=$(hostname)
+        local secondary_host=$(ssh -i /root/.ssh/id_rsa_prvsnr -o "StrictHostKeyChecking no" ${private_data_ip_node_2} "hostname")
+        # echo ${secondary_host}
         local line_to_replace=$(grep -m1 -noP "master: " /etc/salt/minion|tail -1|cut -d: -f1)
         # echo ${line_to_replace}
         
         echo -n "Setting salt-master on server A (primary node).............................." 2>&1|tee -a ${LOG_FILE}
-        sed -i "${line_to_replace}s|^master:.*|master: ${host}|" /etc/salt/minion
+	sed -i "${line_to_replace}s|^master:.*|master: ['${localhost_ip}', '${secondary_host}']|" /etc/salt/minion
         echo "Ok." | tee -a ${LOG_FILE}
 
         echo -n "Setting salt-master on server B (secondary node)............................" 2>&1|tee -a ${LOG_FILE}
-        ssh -i /root/.ssh/id_rsa_prvsnr -o "StrictHostKeyChecking no" ${private_data_ip_node_2} "sed -i \"${line_to_replace}s|^master:.*|master: ${host}|\" /etc/salt/minion"
+	ssh -i /root/.ssh/id_rsa_prvsnr -o "StrictHostKeyChecking no" ${private_data_ip_node_2} "sed -i \"${line_to_replace}s|^master:.*|master: \['${localhost_ip}', '${primary_host}'\]|\" /etc/salt/minion"
```

Comment:
@divya-vijay 
On secondary node this list should be reversed:
`\['${primary_host}', '${localhost_ip}'\]`

Reply:
@divya-vijay 
`salt '*' state.apply components.ha.haproxy.config` mentioned on the ticket is missed.
@83bhp what would be the best place to place this?

Reply:
@ypise Thanks. Yes, I wanted to confirm this placement as well. 
@83bhp We call the haproxy.config states in init script for cluster update, can we call it right after config_update step, as well?
Similar to https://github.com/Seagate/cortx-prvsnr/blob/pre-cortx-1.0/cli/src/factory_ops/unboxing/init#L229